app.py

```diff
@@ -0,0 +1,18 @@
+from flask import Flask, render_template, request
+from attendance import Attendance
+
+app = Flask(__name__)
+attendance = Attendance()
+
+@app.route('/')
+def home():
+    return render_template('index.html', students=attendance.get_students())
```

Comment:
Consider moving the 'attendance' object initialization to the 'home' function. This would ensure a fresh attendance object every time the home page is visited.